src/index.ts

```diff
@@ -0,0 +1,89 @@
+interface Note {
+  id: string;
+  title: string;
+  content: string;
+  createdAt: Date;
+  updatedAt: Date;
+  status: 'pending' | 'completed';
```

Comment:
It would be better to create an enum for these cases. Using string values in code is not a best practice.